dips/dip-10.md

```diff
@@ -101,130 +110,87 @@ struct DiemIdDomainEvent {
+**CommandRequestObject:**
 
-* Receiver Diem ID domain must belong to the Receiver VASP. Otherwise, Receiver VASP must return the `400` error code.
-* Sender Diem ID must have a Diem ID domain that belongs to Sender VASP. If this is not the case, the Receiver VASP should return the `400` error code 
-* Receiver VASP may optionally choose how much information to share based on the sender Diem ID. If Receiver VASP decides to not share data, it should return an empty `User Object` with only `diem_id` field set
-* If user is not found, the VASP should return error code `404`  
+| Field 	    | Type 	     | Required? 	| Description 	           |
+|-------	    |------	     |-----------	|-------------	           |
+| _ObjectType   | str        | Y | Fixed value: `CommandRequestObject`|
+| command_type  | str        | Y | Fixed value: `ReferenceIDCommand`|
+| command       | Command object | Y | The Command to sequence. |
+| cid           | str         | Y            | A unique identifier for the Command. Should be a UUID according to [RFC4122](https://tools.ietf.org/html/rfc4122) with "-"'s included. |
 
-## Pay endpoint
+**CommandObject:**
 
-**TODO: This endpoint is going to be merged into DIP 1**
+| Field 	    | Type 	     | Required? 	| Description 	           |
+|-------	    |------	     |-----------	|-------------	           |
+| _ObjectType   | str    | Y | Fixed value: `ReferenceIDCommand`|
+| sender        | str          | Y            | Sender's full Diem ID |
+| sender_address| str          | Y            | Sender's on-chain address |
+| recipient     | str          | Y            | Receiver's full Diem ID |
+| reference_id  | str          | Y            | Reference ID of this transaction to be included into payment metadata |
 
-**Pay endpoint** is used when a user is willing to make a money transfer.
 
-**Request:**
-```json
-{
-   "sender": "<User Object>",
-   "receiver_id": "<Receiver_Diem_ID>",
-   "amount": "<amount>",
-   "currency": "<currency>"
-}
+The format of the success response is:
 ```
-
-| Field 	   | Type 	     | Required? 	| Description 	           |
-|-------	   |------	     |-----------	|-------------	           |
-|sender        |[User Object](https://github.com/diem/dip/blob/master/dips/dip-10.md#user-object)  | Y            | Information about the sender |
-|receiver_id   |str          | Y            | Diem ID of the receiver |
-|amount        |integer      | Y            | Amount of coins to transfer |
-|currency      |str          | Y            | Name of the coin |
-
-**Response:**
-```json
 {
-   "receiver_address": "<Receiver VASP on-chain account address>",
-   "reference_id": "<Payment_Reference_ID>",
-   "receiver": "<User Object>"
+   "_ObjectType": "CommandResponseObject",
+    "status": "success",
+    "result": {
+	    "_ObjectType": "ReferenceIDCommandResponse",
+	    "recipient_address": "c5ab123458df0003415689adbb47326d",
```

Comment:
should we call this `receiver_address` to echo dip-1? either ways, we should be consistent with `recipient` v.s. `receiver`